content/docs/portals.md

```diff
@@ -44,7 +44,9 @@ A typical use case for portals is when a parent component has an `overflow: hidd
 
 > Note:
 >
-> It is important to remember, when working with portals, you'll need to make sure to follow the proper [accessibility guidelines](/docs/accessibility.html).
+> It is important to remember, when working with portals, that managing keyboard focus becomes very important. You'll need to make sure to follow the proper [accessibility guidelines](/docs/accessibility.html#programmatically-managing-focus).
```

Comment:
First sentence has "important" twice. Second sentence feels a little bulky. Can you combine it? 

Something like 
```
When working with portals, remember that [managing keyboard focus](/docs/accessibility.html#programmatically-managing-focus) becomes very important.
```